src/mesido/workflows/grow_workflow.py

```diff
@@ -179,6 +181,19 @@ def read(self):
         """
         super().read()
 
+        # Error checking:
+        # - installed capacity/power of a heating demand is sufficient for the specified heat
+        #   demand profile
+        for key, vals in self._asset_potential_errors.items():
+            if "heat_demand.power" in key:
+                for val in vals:
+                    print(val)
+                logger.error(
+                    "Heating demand insufficient installed capacity: please increase the installed"
+                    " power or reduce the demand profile peak value of the demands listed."
+                )
+                exit(1)
```

Comment:
this part, specifically the exit should only be entered in case len(vals)>1

Reply:
Silly mistake. Thanks I will update

Reply:
Code has been updated